pkg/vici/messages/list_sas.go

```diff
@@ -1,7 +1,7 @@
 package messages
 
 type ListSAS struct {
-	ReKeyTime int64  `vici:"rekey_time"`
+	ReKeyTime int64  `vici:"rekey-time"`
```

Comment:
Why is it changing to `-` instead of `_`? It looks like StrongSwan uses the underscore: https://docs.strongswan.org/docs/5.9/config/rekeying.html#_settings

I'm not sure if this needs to be a match but for style consistency it makes sense that it is.

Reply:
unfortunately this isn't a 1:1 mapping based on the config you can view all the vici message and their arguments here https://github.com/strongswan/strongswan/blob/master/src/libcharon/plugins/vici/README.md#list-sa

but if you look at this [one](https://github.com/strongswan/strongswan/blob/master/src/libcharon/plugins/vici/README.md#list-sa) specifically you can see they use -

Reply:
you're correct. I see it in the other struct below as well.